CHANGELOG.md

```diff
@@ -2,10 +2,42 @@
+
+However, because the polyfills create or extend global objects, you might have other code in your service unintentionally relying on the inclusion of these polyfills. You might need to introduce your own polyfills or rewrite your JavaScript to avoid using the polyfilled features.
+
+### New features
+
+#### Options to add classes to parts of the component
+
+Use these new options to add your own classes to parts of the component:
+- `inputClasses` for the `input` element
+- `hintClasses` for the suggestion hint (`input` element appearing when the text typed by the user matches the start of an option)
+- `menuClasses` for the `ul` element listing the options
+
+You can also add custom classes to the `ul` listing the options through the new `menuAttributes` option, providing a `className` or `class` property.
+
+This change was introduced in:
+- [pull request #602: Add `inputClasses` option](https://github.com/alphagov/accessible-autocomplete/pull/602). Thanks to @andreyyudin for contributing this change.
+- [pull request #649: Update options for styling component inputs](https://github.com/alphagov/accessible-autocomplete/pull/649)
+- [pull request #650: Update ways of configuring menu attributes](https://github.com/alphagov/accessible-autocomplete/pull/650)
+
+#### New class on the component's status
+
+Use the new `autocomplete__status` class in your CSS (or `<CSS_NAMESPACE>__status` if you set [the `cssNamespace` option](https://github.com/alphagov/accessible-autocomplete?tab=readme-ov-file#cssnamespace-default-autocomplete)) to customise the styles of the component's status element (hidden element that makes announcements to assistive technologies).
+
+This change was introduced in [pull request #453: Add className attribute to status component](https://github.com/alphagov/accessible-autocomplete/pull/453) (merged via [pull request #620](https://github.com/alphagov/accessible-autocomplete/pull/620)). Thanks to @lennym for contributing this change.
+
+### Fixes
+
+- [Pull request #496: Handle query change when the text length does not change](https://github.com/alphagov/accessible-autocomplete/pull/496) (merged through [pull request #621](https://github.com/alphagov/accessible-autocomplete/pull/621)). Thanks to @archferns for contributing this change.
+- [Pull request #591: Add `menuAttributes` to fix Axe's 'aria-input-field-name' error](https://github.com/alphagov/accessible-autocomplete/pull/591). Thanks to @mchughbri for contributing this change.
+- [Pull request #600: Fix null aria attributes](https://github.com/alphagov/accessible-autocomplete/pull/600). Thanks to @mark-roberts-ho for contributing this change.
+- [Pull request #606: Correct description of tStatusSelectedOption i18n option in README and tweak some wording for consistency with other parts of the README](https://github.com/alphagov/accessible-autocomplete/pull/606). Thanks to @ellamdav for contributing this change.
```

Comment:
This is a docs fix, rather than a code fix, but it felt important to acknowledge that kind of contribution as well.